Add ChromatiCraft support

KuroPeach opened the pull request:
Add ChromatiCraft support

KuroPeach commented:
Amazon server timedout ：）
![H~0B{ADE9XGZKN@{E`K97WR](https://user-images.githubusercontent.com/19620848/213910416-16a3add6-8525-4b79-93f3-8b02236ba067.png)

Dream-Master commented:
Timeout can fixed with update buildscript and forge gradle to version .13

Dream-Master commented:
The bigger question is why? We don’t have chromatic craft in the pack

KuroPeach commented:
Just for fun, players will occasionally add some extra mods

boubou19 commented:
i'm not a fan of adding extra compat for a mod we intend to use in the pack only. Adding extra compat means more code to maintain, which isn't something we want if we don't intend to provide compat outside of the pack.

Glease reviewed (changes requested):
The added code is tiny. Not too much to maintain honestly, if any at all. Still I'd like a line of comment somewhere saying `this is non-supported content. if this ever errors out in some way feel free to remove this.`

Also please fix the build. You should be able to do it by running `gradlew updateBuildscript` repeatedly and commit all files added/changed by that.

KuroPeach commented:
Now updated build.grade and spotless code

Dream-Master commented:
to continue or review this you need to resolve conflicts @KuroPeach

KuroPeach committed:
update

update

KuroPeach commented:
Re-pulled the master branch

Dream-Master commented:
@KuroPeach can you change/add what glese suggest ? then we can review it

KuroPeach committed:
add additional statement

/** ChromatiCraft is non-supported content. if this ever errors out in some way feel free to remove this.*/

KuroPeach committed:
align

align

KuroPeach commented:
> @KuroPeach can you change/add what glese suggest ? then we can review it

done ：）

Dream-Master commented:
i send gleese a pm so we need to wait.
